rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2017-09-10
```

Comment:
I'd prefer not to pin this to a specific toolchain version.

Reply:
This doesn't preclude the library from being built with a different toolchain if used as a dependency. 
I can remove it, but I still feel there should be a nicer way than `git log` to let a future maintainer know that at some point, this was building fine.